corehq/apps/registration/templates/registration/base.html

```diff
@@ -2,25 +2,10 @@
 
 {% block login-content %}
     <div class="row">
-    {% block breadcrumbs %}
```

Comment:
@amsagoff I took the liberty of dropping the breadcrumbs because this is no longer a three-step process and they felt heavy. Easy to put them back if you think they'll still add value.